packages/library/README.md

```diff
@@ -18,7 +18,7 @@ If you build JavaScript applications on Solana, it’s likely you’ve worked wi
 
 Here’s an example of a common code snippet from `@solana/web3.js`:
 
-```tsx
```

Comment:
Were these like this for a reason, @buffalojoec, @lorisleiva, @mcintyre94?

Reply:
We're all too react pilled.